packages/react/src/components/builder-block.component.tsx

```diff
@@ -664,12 +664,12 @@ export class BuilderBlock extends React.Component<
     );
   }
 
-  get id() {
+  get id(): string {
     const { block } = this;
-    if (!block.id!.startsWith('builder')) {
+    if (block.id && !block.id.startsWith('builder')) {
```

Comment:
Not this?
```suggestion
    if (block.id?.startsWith('builder')) {
```

Reply:
we need to add `builder-` only if its _not_ there.